setup.py

```diff
@@ -117,6 +117,7 @@
     "parameterized",
     "protobuf",
     "psutil",
+    "pyaml>=20.4.0",
```

Comment:
This is pyaml: https://github.com/mk-fg/pretty-yaml and not PyYaml: https://github.com/yaml/pyyaml

Is this intended? It seems to bit a slight wrapper around PyYaml, but since we're adding it as a core dependency I want to make sure we're on the safer side of things.

Reply:
Good catch, this is fixed now!